docs/samples/explanation/aif/germancredit/README.md

```diff
@@ -0,0 +1,85 @@
+# Bias detection on an InferenceService using AIF360
+
+## End-to-end bias detection with german-credit dataset
+
+This is an example of how to get bias metrics using AI Fairness 360 (AIF360) on KFServing. We will be using the German Credit dataset maintained by the [UC Irvine Machine Learning Repository](https://archive.ics.uci.edu/ml/index.php). The German Credit dataset is a dataset that contains data as to whether or not a creditor gave a loan applicant access to a loan along with data about the applicant. The data includes relevant data on an applicant's credit history, savings, and employment as well as some data on the applicant's demographic such as age, sex, and marital status. Data like credit history, savings, and employment can be used by creditors to accurately predict the probability that an applicant will repay their loans, however, data such as age and sex should not be used to decide whether an applicant should be given a loan. 
+
+We would like to be able to check if these "protected classes" are being used in a model's predictions. In this example we will feed the model some predictions and calculate metrics based off of the predictions the model makes. These metrics will give insight as to whether or not the model is biased for or against any protected classes. In this example we will look at the bias our deployed model has on those of age > 25 vs. those of age <= 25 and see if creditors are treating either unfairly.
+
+### Create the InferenceService
+
+Apply the CRD
+
+```
+kubectl apply -f bias.yaml
+```
+
+Expected Output
+
+```
+$ inferenceservice.serving.kubeflow.org/german-credit created
+```
+
+### Deploy the message dumper
+
+Apply the message-dumper CRD which will collect the logs that are created when running predictions on the inferenceservice.
```

Comment:
```suggestion
Apply the message-dumper CRD which will collect the logs that are created when running predictions on the inferenceservice. In production setup, instead of message-dumper Kafka can be used to receive payload logs
```